src/pages/landing/weekend-trading/components/_hero.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex, Container, Show } from 'components/containers'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import { Background } from 'components/elements/background-image'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+
+const BackgroundWrapper = styled(Background)`
+    background-color: transparent;
+`
+
+const Wrapper = styled(Container)`
+    justify-content: space-between;
+    background-color: transparent;
+    height: unset;
+    padding-top: 2rem;
+    padding-bottom: 27rem;
+
+    @media ${device.tabletL} {
+        padding-left: 4rem;
+    }
+
+    @media ${device.mobileL} {
+        padding-left: 2rem;
+        flex-direction: column-reverse;
+        justify-content: center;
+    }
+`
+
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 60.5rem;
+    z-index: 1;
+
+    @media ${device.tabletL} {
+        max-width: 55rem;
+        top: 280px;
+        padding: 0 16px;
+    }
+    @media ${device.mobileL} {
+        max-width: 328px;
+    }
+    @media ${device.mobileM} {
+        padding: 0;
+    }
+`
+
+const HeroContent = styled(Flex)`
+    flex-direction: column;
+    justify-content: flex-start;
+    height: unset;
+
+    ${Header} {
+        font-size: 24px;
+        font-weight: 200;
+        color: var(--color-black-3);
+        display: flex;
+        margin-top: 1rem;
+        line-height: 2.6rem;
+        max-width: 90%;
+    }
+    @media ${device.laptopM} {
+        ${Header} {
+            font-size: 22px;
+        }
+    }
+    @media ${device.tabletL} {
+        ${Header} {
+            font-size: 16px;
+            max-width: 100%;
+        }
+    }
+    @media ${device.mobileL} {
+        ${Header} {
+            font-size: 20px;
+            line-height: 25px;
+            margin-top: 1px;
+        }
+    }
+    @media ${device.mobileS} {
+        ${Header} {
+            max-width: 98%;
+        }
```

Comment:
For the `Headers`, we should be able to use the `type` prop for responsive sizes. If the design is out of sync with the current specification in `typography`, please let me know or you can speak directly with the designer.